dash_core_components/metadata.json

```diff
@@ -139,73 +139,6 @@
       }
     }
   },
-  "src/components/Confirm.react.js": {
```

Comment:
Why is this getting nuked?

Reply:
It was called `Confirm` at first, but I renamed it to `ConfirmDialog`.

Reply:
Yeah, so that's probably what happened.